atomicapp/nulecule/main.py

```diff
@@ -0,0 +1,234 @@
+# -*- coding: utf-8 -*-
+import anymarkup
+import copy
+import logging
+import os
+import shutil
+import uuid
+from string import Template
+
+from atomicapp.constants import (APP_ENT_PATH, EXTERNAL_APP_DIR,
+                                 GLOBAL_CONF, CACHE_DIR)
+from atomicapp.nulecule.lib import NuleculeBase
+from atomicapp.nulecule.container import DockerHandler
+from atomicapp.utils import Utils
+
+logger = logging.getLogger(__name__)
+
+
+class NuleculeManager(object):
+
+    def __init__(self, image, unpack_path=None):
+        self.image = image
+        self.initialize(unpack_path)
+
+    def initialize(self, unpack_path=None):
+        if unpack_path:
+            self.unpack_path = unpack_path
+        else:
+            self.app_name = '{}-{}'.format(
+                Utils.sanitizeName(self.image), uuid.uuid1())
```

Comment:
so this gives us a unique path for every time we execute, right? 

Is there value in adding the uuid here vs just adding it directly to unpack_path below?